public/index.html

```diff
@@ -56,6 +59,19 @@ <h1 data-translation-id="page_title" class='h1'>Twin Cities Aid Map</h1>
     <p class='p txt-small bold'><span data-translation-id="project_data">See this data in</span> <a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vSfnmh9CmtVoBMy2k_-NMhhvF7ursBs2L5IPF9VZ1J-l9P71vjDFw7y6Y6E98d5lNeOAEo7leisP0Y8/pubhtml" target="_blank">Google Sheets</a>.</p>
     <p class='p txt-small bold'><span data-translation-id="project_learn">Learn about this project on</span> <a href="https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/blob/master/README.md" target="_blank">GitHub</a>.</p>
   </div>
+    <div id="help-search" class='help-search padded white-and-blue'>
+    <button id='help-search-close-button' class='help-search-close-button' aria-label="Close help search section">X <span data-translation-id="close">Close</span></button>
+    <h3 class='txt-summary_large_image' data-translation-id="">Search terms and examples:</h3>
+    <p class='txt-small'>Please note that this is not a complete list of search terms to use, but rather a general guide.</p>
+     <ul class='help-search-list'>
+      <li class='txt-small search-example' data-translation-id=""><b>Zip Code</b> - e.g. '55108'</li>
+      <li class='txt-small search-example' data-translation-id=""><b>City Name </b> - e.g. 'St. Paul'</li>
+      <li class='txt-small search-example' data-translation-id=""><b>Item</b>  - e.g. 'food', 'water', or 'toiletries'</li>
+      <li class='txt-small search-example' data-translation-id=""><b>Building Type</b>  - e.g. 'church', or 'shelter'</li>
```

Comment:
I think the UI mockup and state of the data are somewhat disconnected:
 - We don't currently capture "Building Type" anywhere in the Google Sheet. There might be fragments of this in the note but I don't think this should be advertised as feature of the search.
 - I think "Neighborhood" is probably might be more useful to search for than "City" (debatable)
 - I'm not certain what "Item" refers to in the data. Is this something in the needs section?
 - I don't believe we capture Language Criteria in the Google Sheets.

I think we need to take a step back and make certain that someone on the TCMAP side verifies these, and we try out each of these search criteria and validate that they return useful information.

Reply:
Hi, I'm sort of the defacto head of the data entry crew and haven't done a good job chiming in on the Github side of things yet. Hello. We are SO excited about the prospect of search, even as nothing more than a mobile-friendly version of opening the spreadsheet and doing a ctrl-F, and it looks like y'all have some great stuff in the works.

@jdalt, you're entirely correct about "building type" - it crossed my mind today as not a terrible idea to capture eventually, but I'd like to hear more from map users about how difficult sites are to find. My understanding is that many are NOT buildings - a lot of this is tents in parking lots, card tables on lawns, etc.

We...do have a column for neighborhood at the moment and not city (city does exist as part of the address), but...I'm not convinced that neighborhood info we have is (A) accurate or (B) being used for anything anyway.

I just realized how late it is and need to stop there, but if anybody wants to do the official data entry volunteer training and REALLY get a close look at the process, it'll be 6/9 from 19:00-20:30 or so. I'd also be open to running an abbreviated version with folks working specifically on this feature. I'm on OTC Slack as Eileen King.

Reply:
Hey peeps-
I was thinking and curious if you guys feel the same way about this, but I kind of wonder now if we actually don't need to include a search example modal/pop up window. 

It was more of a starting point to provide any help or context but I'm wondering how useful it is now that I'm thinking more on it. I suppose it could help if there are common questions on things but didn't want to feel like we needed to add it if it's unnecessary confusion?

Let me know what you guys think!

Reply:
being that the search isn't as full-featured as, say, google, and folks don't know the parameters they can search, I feel that it would still be nice to add. But I also might be admittedly biased because I spent several hours hacking this out 😅 would like others to weigh in

Reply:
Hi all! Sorry to be so late in weighing in on this, didn't mean to leave you hanging for so long Lanie! 

@laniehei I really appreciate the work you did on this! I think an issue we are facing is that our UI design got ahead of our actual user-driven needs. I think before we add something like this to the design, we need to get user feedback to determine if it is actually needed, and if so, what would make it the most helpful. We are stuck right now at this "user feedback" issue, in that we currently have no good way of getting that feedback unfortunately. So that has made it difficult to weigh in on this design change. 

I'm inclined to agree that explaining that this is an exact search, and not as robust as say google search, could be helpful. But this is also something that takes up real-estate in a fixed part of the screen, and we are currently having conversations about the UI design in general to determine if other things need to be prioritized into that space, such as the 'search' bar itself, which currently is lost when users scroll through locations. 

I think we need to table this design direction for now until we've gotten more feedback.